nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/logback.xml

```diff
@@ -119,7 +119,7 @@
     <logger name="org.apache.nifi.processors.standard.LogAttribute" level="INFO"/>
     <logger name="org.apache.nifi.processors.standard.LogMessage" level="INFO"/>
     <logger name="org.apache.nifi.controller.repository.StandardProcessSession" level="WARN" />
-
+    <logger name="org.apache.parquet.hadoop.InternalParquetRecordReader" level="WARN" />
```

Comment:
Can we add a comment here as to why we're changing this level (i.e. highly unnecessary INFO-level logging done by this class)?